ls1intum/Artemis
# Score, Participation and Submission pages for exam exercises

### Checklist

### Description
Exam exercises have more of the same buttons their course counterparts already had.
Assessment related buttons and features dont work yet.

### Steps for Testing
1. Create exam with exam groups and exercises.
2. Check scores, participations and submissions. (Works best with participations on the exercises)
2.1 Submissions and Participations can be created in Artemis now.
2.2 Edit results in db Bsp: UPDATE artemis.result SET result_string = '1 of 2 points', completion_date = '2020-06-25 18:52:01.000000', score = '50' WHERE (id = '14');
3. Check if course exercises still function without exceptions.

### Screenshots
![grafik](https://user-images.githubusercontent.com/57408107/85761486-17378e80-b713-11ea-849c-2fe6e1258843.png)
